### PR TITLE
tic tac toe game withouth rspec

### PR DESCRIPTION
**Pull request by Jeko0**

### CODE REVIEW

File: week6_tictactoe/lib/game.rb [hunk 1]
```diff
@@ -0,0 +1,81 @@
...
+  end
+
+  private
+
+  def make_move(board, row, col, turn)
+    print turn == @@PLAYER_ONE ? "First player's turn: " : "Second player's turn: "
+    loop do
+      move = gets.chomp.upcase.chars.sort
+      if board[row.index(move[0])][col.index(move[1])].nil?
+        board[row.index(move[0])][col.index(move[1])] = (turn == @@PLAYER_ONE ? 'X' : 'O')
```
Comment (alexshvedko) [inline week6_tictactoe/lib/game.rb]: nested logic

File: week6_tictactoe/lib/game.rb [hunk 2]
```diff
@@ -0,0 +1,81 @@
...
+    turn = @@PLAYER_ONE
+
+    puts "\nLet's Play Tic-Tac-Toe\n"
+    implement_board(board, row_index, col_index)
+
+    REPEATE_COUNT.times do
+      make_move(board, row_index, col_index, turn)
+      implement_board(board, row_index, col_index)
+      if game_won?(board, turn)
+        puts "Congratulations. Player #{turn == @@PLAYER_ONE ? 'one' : 'two'} won!"
```
Comment (alexshvedko) [inline week6_tictactoe/lib/game.rb]: nested logic

File: week6_tictactoe/lib/game.rb [hunk 3]
```diff
@@ -0,0 +1,81 @@
...
+    row_win = board
+    col_win = board.transpose
+    diagonal_win = [[board[0][0], board[1][1], board[2][2]], [board[0][2], board[1][1], board[2][0]]]
+    wins = row_win + col_win + diagonal_win
+
+    wins.each { |try| return true if try.uniq.length == 1 && !try[0].nil? }
+    false
+  end
+
+  def play
```
Comment (alexshvedko) [inline week6_tictactoe/lib/game.rb:59]: big method

File: week6_tictactoe/lib/game.rb (hunk 2 above)
Comment (Jeko0) [reply week6_tictactoe/lib/game.rb]: fixed

File: week6_tictactoe/lib/game.rb (hunk 3 above)
Comment (Jeko0) [reply week6_tictactoe/lib/game.rb:59]: fixed

File: week6_tictactoe/lib/game.rb (hunk 1 above)
Comment (Jeko0) [reply week6_tictactoe/lib/game.rb]: need help with this one :)

**Pull request merged**